preview.hpp

```diff
@@ -24,6 +25,7 @@ class Preview
 	{
 		done_callback_ = callback;
 	}
+	virtual void SetInfoText(const std::string &text) = 0;
```

Comment:
I fine either way with this one, but given that 2 classes define an empty implementation for this method, maybe we should just supply that by default? Don't really mind though.

Reply:
Yes, makes sense